src/walletSdk/Horizon/Stellar.ts

```diff
@@ -73,4 +81,21 @@ export class Stellar {
       throw e;
     }
   }
+
+  async submitWithFeeIncrease(
```

Comment:
@Ifropc I simplified a bit from what [kotlin is doing](https://github.com/stellar/kotlin-wallet-sdk/blob/77e4bfe0eae19b2ede5af231147351f8367cef01/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/horizon/Stellar.kt#L161-L230) here. For two reasons:

1. keeps it consistent with how the regular `submitTransaction` works
2. there's not a great way in typescript to do the lambda function thing that kotlin is doing [here](https://github.com/stellar/kotlin-wallet-sdk/blob/77e4bfe0eae19b2ede5af231147351f8367cef01/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/horizon/Stellar.kt#L203) for `buildingFunction`. In Typescript we'd have to create a new TransactionBuild object w/in the function, which would be a bit extra complexity we wouldn't need imo (Unless there's a reason `submitWithFeeIncrease` needs to create a new transaction)

wdyt?

Reply:
Current implementation doesn't match Kotlin one.
There is reason it needs to create a new transaction. It bumps fee on every step. 
So, first transaction is submitted with a fee of 100
Next transaction is submitted with a fee of 11000 and so on (assuming fee increase is 1000)
Without this mechanism transaction submitting is not reliable during network surges

Reply:
BTW, we should also add maxFee to this method

Reply:
yup I see that now, totally whiffed on that

fixed with adding maxFee: https://github.com/stellar/typescript-wallet-sdk/pull/31/commits/bbd0f8753535d4ccd5208a801943ced3ad7df5c0